### DISCUSSION TITLE
a link to this "forum" in homepage

### DISCUSSION THREAD

**Discussion by atesin** [nesbox/TIC-80#2193] [Ideas]
hi... i think forums are really useful to share ideas and agilize community and development

but im afraid this "forum" is a little hidden and unknown....  would be great if a link would put in the homepage (tic80.com) to make it more spread and promoted...  turning the project more dynamic too

**Comment by Anrock** [comment]
The link is in the footer

**Comment by atesin** [comment]
yes, the link to github project homepage... but i mean a direct link to discussions (this section)

**Comment by nesbox** [comment]
Added "Forum" to the tic80.com footer
![image](https://github.com/nesbox/TIC-80/assets/1101448/c98de8f9-d9c7-4d01-87d2-b2c0681a5e74)

**Comment by atesin** [comment]
hi @nesbox ... that was fast

first place, i know you added the link and closed the issue....  but i had the feeling there is another "forum" in itch.io page ..  and it was

- https://nesbox.itch.io/tic80/community

...  until it seem outdated...  take this just as a remainder :)

according itch.io page, i didnt know there is a discord group (outdated?) and a vkontakte page, until vk is not so popular over here (chile) (anyway i hate social networks xD)
